### PR TITLE
[mesh] canceling the unnecessary goroutine using

### PR DESCRIPTION
**Pull request by benja-wu**

### CODE REVIEW

**Review by megaeasex — APPROVED** [review]
[TASK:easegress-pr-test SUCCESS]megaease/easegress Pull Request 274 Deploy Test Success

File: pkg/object/meshcontroller/master/master.go [hunk 1]
```diff
@@ -79,7 +79,7 @@ func New(superSpec *supervisor.Spec) *Master {
 	}
 	m.maxHeartbeatTimeout = heartbeat * 2
 
-	go m.run()
+	m.run()
```
Comment (xxx7xxxx) [inline pkg/object/meshcontroller/master/master.go:82]: Why block here, would not it return master？

**Review by localvar — APPROVED** [review]

File: pkg/object/meshcontroller/master/master.go (hunk 1 above)
Comment (benja-wu) [reply pkg/object/meshcontroller/master/master.go:82]: The implementation of `run` is asynchronous. So it won't be blocked here.

Comment (xxx7xxxx) [reply pkg/object/meshcontroller/master/master.go:82]: Then this code isn't basing on the latest code..

Comment (xxx7xxxx) [reply pkg/object/meshcontroller/master/master.go:82]: Then this code isn't basing on the latest code..

Comment (benja-wu) [reply pkg/object/meshcontroller/master/master.go:82]: Yes, I found that too. It's not based on the latest code. Let me fix it. Sorry

**Review by localvar — APPROVED** [review]

File: pkg/object/meshcontroller/master/master.go (hunk 1 above)
Comment (benja-wu) [reply pkg/object/meshcontroller/master/master.go:82]: Fixed. @xxx7xxxx

**Review by megaeasex — APPROVED** [review]
[TASK:easegress-pr-test SUCCESS]megaease/easegress Pull Request 274 Deploy Test Success

**Review by xxx7xxxx — APPROVED** [review]

**Pull request merged**